app/controllers/tasks_controller.rb

```diff
@@ -1,6 +1,16 @@
 class TasksController < ShipitController
+  include Pagination
+
   before_action :stack
 
+  self.default_page_size = 20
+
+  def index
+    @paginator = paginate(stack.deploys)
```

Comment:
This variable is not being used in any view so I think we can change it to be a local variable instead a instance variable.